src/main/java/ch/njol/skript/lang/Expression.java

```diff
@@ -30,7 +33,7 @@
  * @see SimpleExpression
  * @see SyntaxElement
  */
-public interface Expression<T> extends SyntaxElement, Debuggable, Loopable<T> {
+public interface Expression<T> extends SyntaxElement, Debuggable, Loopable<T>, Simplifiable<Expression<? extends T>> {
```

Comment:
Do we want the breaking change of Expressions not being Simplifiable by default? It makes sense for classes like PropertyExpression or WrapperExpression though.

Reply:
Do you think that's beneficial? I didn't make it a breaking change since I didn't really see a reason to do so.

Reply:
Given that we have the interface, it seems like something to just implement if you need it. I don't have a strong opinion either way though.

Reply:
I suppose that makes sense, though as a counterpoint, having simplify() being available as a natural override may make it more available for contributors, so they remember it exists. I also don't have a strong opinion